sdk/core/azure_core/src/http/pager.rs

```diff
@@ -314,7 +318,7 @@ impl<P: Page> ItemIterator<P> {
     /// }
     /// let url = "https://example.com/my_paginated_api".parse().unwrap();
     /// let mut base_req = Request::new(url, Method::Get);
-    /// let pager = ItemIterator::from_callback(move |next_link: PagerState<Url>, ctx| {
+    /// let pager = ItemIterator::from_callback(move |next_link: PagerState<Url>, ctx: Context| {
```

Comment:
Nit: not really necessary since it can be inferred. Most times, `next_link` can as well but I know that was already there and probably because that couldn't be inferred by code therein.

Reply:
I added it because next_link (which can also be inferred) had an explicit type specified and thought that it made sense to document the type of both.